apis/python/setup.py

```diff
@@ -162,9 +167,9 @@ def run(self):
 
 
 INC_DIRS = [
-    "../../libtiledbsoma/include",
-    "../../libtiledbsoma/external/include",
-    "../../build/externals/install/include",
+    "dist_links/libtiledbsoma/include",
+    "dist_links/libtiledbsoma/external/include",
+    "../../build/externals/install/include",  # FIXME
```

Comment:
Here's where I got stuck -- this CMake build directory isn't bundled into the sdist tarball, and I think it'd be weird to do so. While building from the sdist, CMake should generate a similar `build` directory, just in some location other than `../..`. We can figure out what location to expect it in that case and branch this accordingly.

However, I have to say reaching into CMake's stuff from here feels a little weird to me, and I wish I understood better why it's needed now [but wasn't previously](https://github.com/single-cell-data/TileDB-SOMA/commit/123cb4370057eb47c5657ca2f846336a86e6032e#diff-66aed70e28174320f1db4eaccedcab15d85d14b96fb0c2e9f03417e6fc07df92L196)...

Reply:
We moved the Pybind11 code from `libtiledbsoma/src/pyapi` into `apis/python/src/libtiledbsoma` so that the Pybind11 bindings reside with the Python code instead of with the C++ code. When I was working on the conda builds, there were issues with `import tiledbsoma` not being able to find the `pytiledbsoma.so` Pybind11 generated shared library. We decided use a working model (TileDB-Py) and reorganized the code to get the conda builds working.